securedrop/sass/source.sass

```diff
@@ -143,6 +143,7 @@ p#codename
     margin-bottom: 1em
     border-radius: 5px
     position: relative
+    word-wrap: break-word
```

Comment:
According to <https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap>, the official name is "overflow-wrap" and "word-wrap" is an alias for legacy reasons, could you switch to overflow-wrap?

Reply:
Sure, I can switch "word-wrap" to "overflow-wrap".